pkg/disk/disk_unix.go

```diff
@@ -13,13 +13,13 @@ import (
 	"io/fs"
 	"path"
 
+	"github.com/docker/go-units"
 	limaStore "github.com/lima-vm/lima/pkg/store"
 )
 
 const (
 	// diskName must always be consistent with the value under additionalDisks in finch.yaml.
```

Comment:
nit: maybe we can modify this to keep the name consistent with value in lima_config_applier.go?

Reply:
Not sure what you mean, can you put a code suggestion?

Reply:
I meant the comment, since finch.yaml does not define additionalDisks and it's changed to https://github.com/runfinch/finch/pull/594/files#diff-0cc05068a0ab38b109e62d43a953ab4a2d8563df608ab52f3a12d0933e1e3a9fR146. 
```suggestion
	// diskName must always be consistent with the value under lima_config_applier.go 
```

Reply:
Oh, got it. Fixed in latest revision